### ISSUE TITLE
`process` variable clobbers environment

### ISSUE THREAD

**Issue by lobsteropteryx** [18F/web-design-standards#1067]
In some of the gulp task files (`release.js` and `website.js`), `child_process` is being brought in like so:

```
var process = require('child_process');
```

This is clobbering the existing `process` [object](https://nodejs.org/api/process.html).  At the very least, this means that `process.env` is undefined when spawning child processes.

I'm not sure what is idiomatic here, but I think something like

```
var child_process = require('child_process');
```

might be safer.  

I've tested this change locally, and will submit a PR.

@jonahadkins

**Comment by rogeruiz** [comment]
Thanks @lobsteropteryx for this. A PR updating the variables to

```
var spawn = require('child_process').spawn;
var exec = require('child_process').exec;
var execSync = require('child_process').execSync;
``` 

might be the cleaner route than using `child_process` as the variable isn't being used for anything other than getting to the `spawn`, `exec`, and `execSync` commands. Node will cache the `require('child_process')` calls.

**Comment by rogeruiz** [comment]
Fixed by #1073
